config/index.js

```diff
@@ -184,6 +184,11 @@ var conf = convict({
       format: String,
       default: 'https://www.mozilla.org/firefox/ios/'
     },
+    signInUrl: {
+      doc: 'Signin url',
+      format: String,
+      default: 'https://accounts.firefox.com/signin'
```

Comment:
This should be like the rest of the urls below, see https://github.com/mozilla/fxa-auth-server/blob/master/config/index.js#L367